src/ayb_db/models.rs

```diff
@@ -166,13 +166,27 @@ pub struct InstantiatedDatabase {
 pub struct Entity {
     pub slug: String,
     pub entity_type: i16,
+    pub display_name: Option<String>,
+    pub description: Option<String>,
+    pub workplace: Option<String>,
+    pub links: Option<Vec<Link>>,
```

Comment:
sanity-checking only the instantiated one should have the sqlx JSON wrapper

Reply:
Exactly :)